source/vfs/vfs.go

```diff
@@ -0,0 +1,137 @@
+// Package vfs contains a driver that reads migrations from a virtual file
+// system.
+//
+// Implementations of the filesystem interface that read from zip files and
+// maps, as well as the definition of the filesystem interface can be found in
+// the golang.org/x/tools/godoc/vfs package.
+package vfs
```

Comment:
Rename package (and directory) to `go_vfs`

Reply:
Done